SPECS/grpc/grpc.spec

```diff
@@ -1,14 +1,15 @@
 Summary:        Open source remote procedure call (RPC) framework
 Name:           grpc
 Version:        1.42.0
-Release:        6%{?dist}
+Release:        7%{?dist}
 License:        ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Applications/System
 URL:            https://www.grpc.io
 Source0:        https://github.com/grpc/grpc/archive/v%{version}/%{name}-%{version}.tar.gz
 Source1:        %{name}-%{version}-submodules.tar.gz
+Patch0:         CVE-2023-4785.patch
```

Comment:
I do not think, we leverage `%autsetup` for this particular spec.
Either we could add `autosetup` or `%patch0 -p1`. My bias would be towards `%autosetup` though :)